src/pygama/evt/tcm.py

```diff
@@ -42,11 +44,9 @@ def generate_tcm_cols(
     window_ref
         when testing one datum for inclusion in a cluster, test if it is within
         `coin_window` of
-
```

Comment:
without newlines before and after lists these will not render properly on the web docs

Reply:
the Build Documentation check made the same complaint